istioctl/cmd/istioctl/deregister.go

```diff
@@ -25,12 +25,15 @@ var (
 	deregisterCmd = &cobra.Command{
 		Use:   "deregister <svcname> <ip>",
 		Short: "De-registers a service instance",
-		Args:  cobra.MinimumNArgs(2),
+		Example: `# de-register an endpoint 172.17.0.2 from service my-svc:
+istioctl deregister my-svc 172.17.0.2`,
+		Args: cobra.MinimumNArgs(2),
 		RunE: func(c *cobra.Command, args []string) error {
 			svcName := args[0]
 			ip := args[1]
 			log.Infof("De-registering for service '%s' ip '%s'",
 				svcName, ip)
+			// TODO: Support deregister endpoint from service registry other than Kubernetes.
```

Comment:
I believe the long term plan is to use `ServiceEntry` for this purpose rather than creating k8s service/endpoints directly. In that case, `register` / `unregister` would be deprecated. cc @costinm

Reply:
Okay, thanks @ayj 

So, I should remove the TODO line?